src/main/java/com/softserveinc/ita/pageobjects/HomePage.java

```diff
@@ -40,22 +42,20 @@ public ProductPage openLastProductOfSection(String sectionName) {
         return new ProductPage();
     }
 
-    public String homePageLastViewedProductTitle(String expectedItem) {
+    public List<String> homePageLastViewedProductTitle(String expectedItem) {
```

Comment:
I mean something else, rename and do not pass anything, you just need from these screen return list of items titles

Reply:
and do some magic in test

Reply:
you mean:
   public ElementsCollection selectFirstItemFromLastProduct() {
        return listOfLastViewedItems
                .shouldBe(sizeNotEqual(0), ofSeconds(8)); ???

and in test i have to stream and filter by name, right?